jsign-crypto/src/main/java/net/jsign/jca/VenafiCredentials.java

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
I changed the license headers two days ago, they now start with `/*` instead of `/**`